migration-util/src/main/java/edu/unc/lib/dcr/migration/content/ACLTransformationHelpers.java

```diff
@@ -195,8 +200,10 @@ private static void mergeParentPatronAcls(PID parentPid, Resource destResc, Prop
         if (parentUnitModel != null) {
             Resource parentUnitResc = parentUnitModel.getResource(parentPid.getRepositoryPath());
             if (!destResc.hasProperty(CdrAcl.embargoUntil) && parentUnitResc.hasProperty(CdrAcl.embargoUntil)) {
-                destResc.addLiteral(CdrAcl.embargoUntil,
-                        parentUnitResc.getProperty(CdrAcl.embargoUntil).getLiteral().getString());
+                String embargoDate = formatEmbargoDate(parentUnitResc.getProperty(
```

Comment:
Since this is just carrying over an embargo from a parent object that already went throught the formatting method, I think you can skip most of these steps and just do something like `destResc.addLiteral(CdrAcl.embargoUntil, parentUnitResc.getProperty(CdrAcl.embargoUntil).getObject())`. If that doesn't work, you can probably at least skip the formatEmbargoDate call here